### PR TITLE
fix(TUP-33414): [bug] JETL: Studio is closed after patch installed

### PR DESCRIPTION
**Pull request by bhe-talendbj**

### CODE REVIEW

File: main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java [hunk 1]
```diff
@@ -115,6 +151,13 @@ public static String getEclipseArgument(String argName) {
         if (argName == null || argName.trim().isEmpty()) {
             return null;
         }
+
+        if (isWindows() && TALEND_ARGS.contains(argName) && !isPoweredByTalend()) {
+            ExceptionHandler.logDebug("argName: " + argName + ", sysProp: " + argName + ", value: " + System.getProperty(argName));
+            Properties p = loadConfigIni();
+            return p.getProperty(argName);
+        }
```
Comment (cmeng-talend) [inline main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java:159]: maybe it's better to add a try catch here

File: main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java [hunk 2]
```diff
@@ -128,6 +171,152 @@ public static String getEclipseArgument(String argName) {
         return null;
     }
 
+    private static boolean isPoweredByTalend() {
+        return !Platform.getProduct().getId().equals("org.talend.rcp.branding.jetl.product") && !Platform.getProduct().getId().equals("org.talend.rcp.branding.jetl.bigdata.product");
+    }
+    
+    private static boolean isWindows() {
+        return System.getProperty("os.name").startsWith("Windows");
```
Comment (cmeng-talend) [inline main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java:179]: maybe can reuse the EnvironmentUtils.isWindows here

Comment (bhe-talendbj) [reply main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java]: done

File: main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java (hunk 1 above)
Comment (bhe-talendbj) [reply main/plugins/org.talend.commons.runtime/src/org/talend/commons/utils/system/EclipseCommandLine.java]: done

**Review by nrousseau — APPROVED** [review]

**Pull request merged**